### PR TITLE
Adding topic qt

### PR DESCRIPTION
**Pull request by himanshu1603**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by cheshire137** [comment]
Thank you for the PR! You have a couple failing tests: 

> logo should be qt.png, but was Qt_logo.png

> topic images should be 288px wide.
Expected: 288
  Actual: 87

Can you get a larger, square logo, size 288x288, and rename it to `qt.png`?

File: topics/data-structure/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: data-structures
+display_name: Data structure
+short_description: Data structure is way of organizing and storing data.
+topic: data-structure
```
Comment (cheshire137) [inline topics/data-structure/index.md]: `data-structures` is listed in https://github.com/github/explore/blob/master/topics-todo.md as one of the topics we want to curate, so thank you! Would you mind renaming this directory to data-structures/ and make `data-structure` an alias of `data-structures`? Then this line can be `topic: data-structures` and you can check off `data-structures` in the topics-todo.md file in your branch.

File: topics/data-structure/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: data-structures
+display_name: Data structure
+short_description: Data structure is way of organizing and storing data.
```
Comment (cheshire137) [inline topics/data-structure/index.md]: How about:

> Data structures are a way of organizing and storing data.

File: topics/data-structure/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+topic: data-structure
+wikipedia_url: https://en.wikipedia.org/wiki/Data_structure
+---
+ Data structure is a particular way of organizing and storing data in a computer so that it can be accessed and modified efficiently.It is designed to organize data to suit a specific purpose so that it can be accessed and worked within appropriate ways.Examples: array,linked list, class etc.
```
Comment (cheshire137) [inline topics/data-structure/index.md]: Can you remove the leading space?

It looks like this is copied from the Wikipedia page. Maybe we can rephrase? How about:

> A data structure is a particular way storing and organizing data in a computer for efficient access and modification. Data structures are designed for a specific purpose. Examples include arrays, linked lists, and classes.

File: topics/qt/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+aliases: qt-framework, cute, qt-application
+created_by: Haavard Nord, Eirik Chambe-Eng
+display_name: Qt
+github_url: https://github.com/qt
+logo: Qt_logo.png
+released: 20 May 1995
+short_description: Qt is a cross-platform application framework that is used for developing application software.
+topic: qt
```
Comment (cheshire137) [inline topics/qt/index.md]: I see qt is on the list https://github.com/github/explore/blob/master/topics-todo.md. Would you mind editing topics-todo.md in this branch to check off `qt`? You can replace the line `[ ] qt` with `[x] qt` to do so.

File: topics/qt/index.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+aliases: qt-framework, cute, qt-application
```
Comment (cheshire137) [inline topics/qt/index.md]: Looking at https://github.com/topics/cute, the repositories using that topic don't seem related to Qt. Can you remove `cute` from this list?

I see https://github.com/topics/qt5 is pretty popular. What about adding `qt5` to this list?

File: topics/qt/index.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+created_by: Haavard Nord, Eirik Chambe-Eng
+display_name: Qt
+github_url: https://github.com/qt
+logo: Qt_logo.png
```
Comment (cheshire137) [inline topics/qt/index.md]: This should be `logo: qt.png`.

File: topics/qt/index.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+released: 20 May 1995
```
Comment (cheshire137) [inline topics/qt/index.md]: Can you format this as `May 20, 1995`?

File: topics/qt/index.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+short_description: Qt is a cross-platform application framework that is used for developing application software.
```
Comment (cheshire137) [inline topics/qt/index.md]: This could be rephrased so it isn't the same as the Wikipedia page. What about:

> Qt, pronounced "cute", is a cross-platform application development framework.

File: topics/qt/index.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+topic: qt
+related: cpp
+url: https://www.qt.io/
+wikipedia_url: https://en.wikipedia.org/wiki/Qt_(software)
+---
+Qt is a cross-platform application network that is used for developing application that can run on various hardware and software platforms. Qt is available with both proprietary and open source licenses.
```
Comment (cheshire137) [inline topics/qt/index.md]: > application network

Is this right?

> can run on various hardware and software platforms

You could list a few:

> can run on various hardware and software platforms, including Linux, macOS, iOS, Windows, and Blackberry.

Source: http://wiki.qt.io/About_Qt

**Comment by himanshu1603** [comment]
Thanks for the review! Adding above mentioned changes

**Comment by cheshire137** [comment]
Thanks for the update! The logo looks like the aspect ratio was changed when it was resized to fit 288x288. It would be preferable to omit the logo rather than include one that's scaled up if the aspect ratio changes.

**Comment by himanshu1603** [comment]
@cheshire137  please review these changes too.

File: topics/scikit-learn/index.md [hunk 10]
```diff
@@ -1,14 +0,0 @@
----
```
Comment (MikeMcQuaid) [inline topics/scikit-learn/index.md]: Whoops, did need to undelete this file and those below @himanshu1603!

**Comment by himanshu1603** [comment]
@MikeMcQuaid  please review the changes I've made.

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @himanshu1603!
